mobify/platform-scaffold
# Fix How we're running Tests with Preview on Production

On master branch, the TEST_PROFILE=production was not being set as start-test-server.sh sets it in the background.

Change this by adding a few lines to check for git branch and export the production variable on a foreground process - run-all-tests.sh

Add run-lighthouse.sh script to have logic of when to run test:pwa-ci vs test:pwa-prod

## Changes
- web/tests/system/run-all-tests.sh 
-web/tests/system/run-lighthouse.sh

##TODOs
- Should also merge to develop to update

## How to test-drive this PR
Change the line 10 on run-all-tests.sh 
"[ "$CURRENT_BRANCH" == "master" ]' to the current branch (for testing purposes to force 'production' to get exported) 
- ./web/tests/system/run-all-tests.sh
- Nightwatch should be testing on the production
- Production tests should fail right now due to us not publishing a bundle for v0.17.0
